iothub/service/src/Registry/RegistryManager.cs

```diff
@@ -294,7 +295,7 @@ public virtual Task<Device> AddDeviceAsync(Device device, CancellationToken canc
             finally
             {
                 if (Logging.IsEnabled)
-                    Logging.Exit(this, $"Adding device: {device?.Id}", nameof(AddDeviceAsync));
+                    Logging.Exit(this, $"Adding device {device?.Id} with type {device?.Authentication.Type}", nameof(AddDeviceAsync));
```

Comment:
I removed some test logging where we already had SDK logging. In this case, the SDK logs didn't indicate the auth type, so I added it here and removed the test log.